.github/workflows/pr_dry.yml

```diff
@@ -0,0 +1,136 @@
+name: Pull Request
+
+on:
+  pull_request_target:
+    types: ['opened', 'reopened', 'edited']
```

Comment:
:information_source: I restricted this workflow to only those events to avoid running it on each commit inside a PR.

Editing PR title/description will trigger the workflow though